pkg/etcd/etcd_test.go

```diff
@@ -275,8 +275,7 @@ func Test_UnitETCD_Start(t *testing.T) {
 			setup: func(e *ETCD, ctxInfo *contextInfo) error {
 				ctxInfo.ctx, ctxInfo.cancel = context.WithCancel(context.Background())
 				testutil.GenerateRuntime(e.config)
-				e.runtime = e.config.Runtime
-				client, err := GetClient(ctxInfo.ctx, e.runtime, endpoint)
+				client, err := GetClient(ctxInfo.ctx, e.config.Runtime)
```

Comment:
I think we want to check the error here so we don't assign a nil client below and then we can return nil on 201.

Reply:
This is in the test setup function; I'll defer to @dereknola but it looks like an error returned by any test setup will cause the entire suite to fail early, so the nil client should be OK.